parcel/ParcelTendencies.jl

```diff
@@ -3,6 +3,7 @@ import CloudMicrophysics.Common as CMO
 import CloudMicrophysics.HetIceNucleation as CMI_het
 import CloudMicrophysics.HomIceNucleation as CMI_hom
 import CloudMicrophysics.Parameters as CMP
+using Distributions
```

Comment:
Could you replace `using Distributions` with `import Distributions as DS`. And then replace all the function calls to functions from the `Distributions` module with `DS.foo`. It helps to keep track of which function is defined where.

(here are some docs on it: https://docs.julialang.org/en/v1/manual/modules/#Renaming-with-as)